rust/connlib/clients/apple/build-xcframework-dev.sh

```diff
@@ -0,0 +1,27 @@
+# For more info:
+# https://github.com/firezone/firezone-apple/blob/main/USING_UNRELEASED_CONNLIB.md
+
+#!/bin/bash
+set -ex
+
+echo $SRC_ROOT
+
+for sdk in macosx; do
```

Comment:
Maybe we should make this build for both macOS and iOS by default?
```suggestion
for sdk in macosx iphoneos; do
```

Reply:
Ah, thanks for catching this @roop; yes on Apple I think we decided to develop for both `iphoneos` and `macosx` targets in parallel since the platforms are 90% similar. The main difference is only the UI (and the hurdles involved with debugging on the iphonesimulator)